js2-mode.el

```diff
@@ -161,6 +161,19 @@
             window alert confirm document java navigator prompt screen
             self top requestAnimationFrame cancelAnimationFrame
 
+            ;; Window or WorkerGlobalScope with support in Chromium and Firefox:
+            ;; https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope
+            ;; properties
+            caches indexedDB isSecureContext origin
+            ;; methods
+            atob btoa clearInterval clearTimeout
+            createImageBitmap fetch queueMicrotask setInterval
+            setTimeout
+            
+            ;; from jslint "browser"
+            Audio document event history Image location name
+            navigator Option screen XMLHttpRequest
```

Comment:
`XMLHttpRequest` is specified at the end of the existing list.

`navigator`, `screen`, `document` are also present elsewhere.

Could you maybe find specific sections for `Audio`, `Image` and `Option`?